includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -382,4 +402,23 @@ private function getSitenameData() {
 			'value' => WikiFactory::getVarValueByName( 'wgSitename', $this->productInstanceId, false, $this->wg->Sitename ),
 		];
 	}
+
+	private function getRightSlot() {
+		if ( $this->lang === 'de' ) {
```

Comment:
Yes, Product agreed on using user/browser language to show this logo. It means that we don't need to solve this problem now.